### PR TITLE
[Meetings][Usertypedetails][1677986]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Updated user types details and added table for features as per issue 2995. Please review.

### CODE REVIEW

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 1]
```diff
@@ -139,7 +140,15 @@ The following user types identify what each user can do and what they can access
...
+* **Anonymous**: Anonymous users do not have an AAD identity and are not federated with a tenant. The anonymous participant is like an external user, but their identity is not projected in the meeting. An anonymous user cannot be an organizer but can be a presenter or attendee.
+
+The following table provides the user types and what features each user can access:
+
+| User type | Tabs | Bots | Message extensions | Adaptive cards | Task modules | In-meeting dialog |
+| :-- | :-- | :-- | :-- | :-- | :-- | :-- |
+| Anonymous user | Not available | Not available | Not available | Interactions in the meeting chat are allowed. | Interactions in the meeting chat from an adaptive card are allowed. | Not available |
+| Guest that is part of the tenant AAD | Interaction is allowed. Creating, updating, and deleting are not allowed. | Not available | Not available | Interactions in the meeting chat are allowed | Interactions in the meeting chat from an adaptive card are allowed | Available |
+| Federated | Not available | Not available | Not available | Not available | Not available | Not available. This is planned to be supported in the future. |
```
Comment (rarang) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:151]: Remove this line "This is planned to be supported in the future" under Federated

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ The following table provides the user types and what features each user can acce
 | :-- | :-- | :-- | :-- | :-- | :-- | :-- |
 | Anonymous user | Not available | Not available | Not available | Interactions in the meeting chat are allowed. | Interactions in the meeting chat from an adaptive card are allowed. | Not available |
```
Comment (rarang) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:149]: Note: Anonymous users inherit the global default user-level app permission policy. Refer to the "Manage Apps" section to learn more.

Comment (rarang) [reply msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:149]: Add this on top of the table

**Review by rarang — APPROVED** [review]

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 3]
```diff
@@ -68,11 +69,11 @@ Tabs allow team members to access services and content in a specific space withi
 
 ✔ Permissioned users can add apps while in the meeting.
 
-✔ When loaded in the context of a meeting, apps can leverage the Teams Client SDK to access the `meetingId`, `userMri`, and `frameContext` to appropriately render the experience.
+✔ When loaded in the context of a meeting, apps can leverage the Teams client SDK to access the `meetingId`, `userMri`, and `frameContext` to appropriately render the experience.
 
 ✔ Exporting a result of a survey or poll notifies the users that the results successfully downloaded.
 
-✔ An app is visible in a Teams meeting in the side panel or the in-meeting dialog box. Use the in-meeting dialog to showcase actionable content for meeting participants. *See* [Create Apps for Teams meetings](create-apps-for-teams-meetings.md).
+✔ An app is visible in a Teams meeting in the side panel or the in-meeting dialog box. Use the in-meeting dialog box to showcase actionable content for meeting participants. For more information , see [create apps for Teams meetings](create-apps-for-teams-meetings.md).
```
Comment (KirtiPereira) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:76]: remove extra space '...information , see...'

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 4]
```diff
@@ -139,7 +140,18 @@ The following user types identify what each user can do and what they can access
...
 * **Federated or external**: A federated user is an external Teams user in another organization who has been invited to join a meeting. These users have valid credentials with federated partners and are authorized by Teams. They do not have access to your teams or other shared resources from your organization. Guest access is a better option for external users to have access to teams and channels. For more information, see [manage external access in Teams](/microsoftteams/manage-external-access).
-* **Anonymous**: Anonymous users do not have an AAD identity and are not federated with a tenant. The anonymous participant is like an external user, but their identity is not projected in the meeting. Anonymous users are not able to access apps in a meeting window. An anonymous user cannot be an organizer but can be a presenter or attendee.
+* **Anonymous**: Anonymous users do not have an AAD identity and are not federated with a tenant. The anonymous participant is like an external user, but their identity is not projected in the meeting. An anonymous user cannot be an organizer but can be a presenter or attendee.
```
Comment (KirtiPereira) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:143]: ...an attendee.

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 5]
```diff
@@ -139,7 +140,18 @@ The following user types identify what each user can do and what they can access
...
+
+> [!NOTE]
+> Anonymous users inherit the global default user-level app permission policy. For more information, see [Manage Apps](/microsoftteams/non-standard-users#anonymous-user-in-meetings-access).
+
+The following table provides the user types and what features each user can access:
+
+| User type | Tabs | Bots | Message extensions | Adaptive cards | Task modules | In-meeting dialog |
+| :-- | :-- | :-- | :-- | :-- | :-- | :-- |
+| Anonymous user | Not available | Not available | Not available | Interactions in the meeting chat are allowed. | Interactions in the meeting chat from an adaptive card are allowed. | Not available |
```
Comment (KirtiPereira) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:152]: Adaptive Card

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 6]
```diff
@@ -139,7 +140,18 @@ The following user types identify what each user can do and what they can access
...
+| Guest that is part of the tenant AAD | Interaction is allowed. Creating, updating, and deleting are not allowed. | Not available | Not available | Interactions in the meeting chat are allowed | Interactions in the meeting chat from an adaptive card are allowed | Available |
```
Comment (KirtiPereira) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:153]: missing periods (.)

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@v-rodriguezd - please validate and incorporate the changes.

**Review by surbhigupta12 — APPROVED** [review]
Approved.

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**